src/appConfig.js

```diff
@@ -21,7 +21,7 @@ if (window.location.host === 'streamstats.usgs.gov') {
         configuration.baseurls.StreamStatsMapServices = 'https://gis.streamstats.usgs.gov',
         configuration.baseurls.NSS = 'https://streamstats.usgs.gov/nssservicesv2',
         configuration.baseurls.nssservicesv2 = 'https://streamstats.usgs.gov/nssservicesv2',
-        configuration.baseurls.WaterUseServices = 'https://streamstats.usgs.gov/wateruseservices',
+        configuration.baseurls.WaterUseServices = 'https://test.streamstats.usgs.gov/wateruseservices',
```

Comment:
This is the section that gives the base urls for prod SS. We should probably push these to the prod wateruseservices when we're ready. Otherwise, it'll get migrated with the next migration anyway.